isolator.gemspec

```diff
@@ -20,9 +20,18 @@ Gem::Specification.new do |spec|
   end
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "anyway_config", "~> 1.0"
```

Comment:
I don't think we need a full-featured configuration here, looks like a overhead.
I suggest using module accessors instead, e.g. `Isolator.raise_exceptions` etc

Reply:
I've used config here to allow changing values based on environment

Reply:
I don't think it make sense, these parameters are to be set once.

Let's get rid of this dependency.